cmd/multi_cmd.go

```diff
@@ -37,5 +37,5 @@ func multi(config *conf.GlobalConfiguration) {
 
 	models.RunHooks(bgDB, logrus.WithField("component", "hooks"))
 
-	api.ListenAndServe(l)
+	logrus.WithError(api.ListenAndServe(l)).Fatal("Error listening")
```

Comment:
That is fine, it can return whatever, but the problem is that the call to `Fatal` causes a bad exit code